dev-requirement.pip

```diff
@@ -0,0 +1,10 @@
+# Include base requirements
+-r requirements.pip
+
+# For unit tests
+coverage==3.6
+mock==1.1.2
+nose==1.3.7
+
+# Transitive Dependencies
+funcsigs==0.4
```

Comment:
Comments about why these particular versions are pegged, rather than newest, would be helpful.